### ISSUE TITLE
Text alignment behaves strangely

### ISSUE THREAD

**Issue by aquarat** [johnfercher/maroto#35]
Firstly, I just want to say this is a very cool/useful project :)

I'm trying to build a PDF quote. As part of that I've got a table and some of the items in the table are numbers (others are normal text). The numbers should ideally align to the right.

With debugging turned on and text on the cells set to "Left" on the "Amount" column :
![Screenshot from 2019-07-19 17-47-28](https://user-images.githubusercontent.com/3090997/61548180-59f5de80-aa4d-11e9-9ca9-dd1b9fe2f9e5.png)

With debugging turned on and text on the cells set to "Right" on the "Amount" column:
![Screenshot from 2019-07-19 17-49-11](https://user-images.githubusercontent.com/3090997/61548245-8c074080-aa4d-11e9-9f89-63665dd8a2e0.png)

Header creation code : 

```
			style := maroto.Bold
			m.Row(0, func() {
				m.Col(func() {
					m.Text("Description", &maroto.TextProp{
						Size:  11,
						Style: style,
						Align: maroto.Left,
					})
				})
				m.Col(func() {
					m.Text("", &maroto.TextProp{
						Size:  11,
						Style: style,
						Align: maroto.Left,
					})
				})
				m.Col(func() {
					m.Text("Quantity", &maroto.TextProp{
						Size:  11,
						Style: style,
						Align: maroto.Right,
					})
				})
				m.Col(func() {
					m.Text("Rate", &maroto.TextProp{
						Size:  11,
						Style: style,
						Align: maroto.Right,
					})
				})
				m.Col(func() {
					m.Text("Amount", &maroto.TextProp{
						Size:  11,
						Style: style,
						Align: maroto.Right,
					})
				})
			})

			m.Line(4.0)
			m.Row(2, func() {

			})
```

and row creation code 

```
for _, j := range cc.ExpandedContent {
				m.Row(4, func() {
					m.Col(func() {
						m.Text(j.Item, &maroto.TextProp{
							Size:  size,
							Style: style,
							Align: maroto.Left,
						})
					})
					m.Col(func() {
						if j.Info != "Yes" {
							m.Text(j.Info, &maroto.TextProp{
								Size:  size,
								Style: style,
								Align: maroto.Left,
							})
						}
					})
					m.Col(func() {
						m.Text(strconv.FormatFloat(float64(j.Qty), 'f', 2, 64), &maroto.TextProp{
							Size:  size,
							Style: style,
							Align: maroto.Right,
						})
					})
					m.Col(func() {
						m.Text(formatCurrency(j.Rate), &maroto.TextProp{
							Size:  size,
							Style: style,
							Align: maroto.Right,
						})
					})
					m.Col(func() {
						m.Text(formatCurrency(j.Rate), &maroto.TextProp{
							Size:  size,
							Style: style,
							Align: maroto.Right,
						})
					})
				})
}
```
With debugging turned off :
![Screenshot from 2019-07-19 17-56-41](https://user-images.githubusercontent.com/3090997/61548719-9b3abe00-aa4e-11e9-9faf-22fefd954dd2.png)

**Comment by aquarat** [comment]
The bug is in text.go in function Add.

This seems to fix it :
```
func (self *text) Add(text string, fontFamily Family, fontStyle Style, fontSize float64, marginTop float64, align Align, actualCol float64, qtdCols float64) {
	actualWidthPerCol := self.math.GetWidthPerCol(qtdCols)

	self.font.SetFont(fontFamily, fontStyle, fontSize)

	left, top, _, _ := self.pdf.GetMargins()

	if align == Left {
		self.pdf.Text(actualCol*actualWidthPerCol+left, marginTop+top, text)
		return
	}

	var modifier float64 = 2

	if align == Right {
		modifier = 1
	}

	translator := self.pdf.UnicodeTranslatorFromDescriptor("")

	stringWidth := self.pdf.GetStringWidth(translator(text))
	dx := (actualWidthPerCol - stringWidth) / modifier

	self.pdf.Text(dx+actualCol*actualWidthPerCol+left, marginTop+top, translator(text))
}
```

**Comment by johnferchermeli** [comment]
Dude, I'm so sorry that I took 2 weeks to see your comment. The github didn't notificate me... I will se your PR now.

**Comment by johnfercher** [comment]
Fixed in #36

**Comment by johnfercher** [comment]
You could use the [TableList](https://godoc.org/github.com/johnfercher/maroto#PdfMaroto.TableList) feature to do that. Currently, this feature does almost the same thing which you have done with theses **for's**. The only thing which I see is that TableList doesn't allow is use different aligns per column. You could take your code I put there, would be a very nice feature.

Maybe [TableListProp](https://godoc.org/github.com/johnfercher/maroto#TableListProp) could have a list of alignments :D
